teuthology/suite/run.py

```diff
@@ -27,6 +27,32 @@
 
 log = logging.getLogger(__name__)
 
+def descr_to_yamls(descriptions, suites_path):
```

Comment:
this methods lacks descriptions, and unit tests, could you please add some

Reply:
@kshtsk Please help. I need suggestion how unit test should look like.

Reply:
One test case you've already put to the description, now just turn it into a code.

Reply:
we're trying to use python typing for new code:

```
def yamls_from_desc(descriptions: str, suite_path: str) -> list[str]:
```